postgresql/initdb.d/01_main.sql

```diff
@@ -140,3 +141,26 @@ CREATE TABLE file_event_log (
     success             BOOLEAN,
     error               TEXT
 );
+
+-- This table is used to define events for dataset event logging.
+CREATE TABLE dataset_events (
+    id          SERIAL PRIMARY KEY,
+    title       VARCHAR(64) UNIQUE, -- short name of the action
+    description TEXT
+);
+
+-- These are the default dataset events to log.
+INSERT INTO dataset_events(id,title,description)
+VALUES (10, 'registered', 'A dataset has been registered'),
```

Comment:
i view this as a mapping message, as it maps dataset to files

also here it is referenced as mapping: https://localega.readthedocs.io/en/latest/amqp.html#central-ega-cegatolega-local-ega

i am fine with either, but i would prefer to use the same terminology

Reply:
maybe we can keep registered and have something like `VALUES (10, 'registered', 'Register a dataset to file accession IDs mapping.'),`